kbchat/kbchat_test.go

```diff
@@ -3,53 +3,52 @@ package kbchat
 import (
 	"crypto/rand"
 	"encoding/hex"
-	"flag"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"os/exec"
 	"path"
+	"strconv"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/require"
 	"gopkg.in/yaml.v2"
 )
 
-type Bots struct {
-	Alice1   *OneshotOptions
-	Alice2   *OneshotOptions
-	Bob1     *OneshotOptions
-	Charlie1 *OneshotOptions
+type botConfig struct {
```

Comment:
i don't love the name of this object. it's intended to map perfectly to the test config that includes all the bots and teams that are made by anyone who wants to run these, yeah? maybe this should be called something like `KeybaseTestData`.

Reply:
i wouldn't mind if we pushed a lot of this stuff into another file. not the testSetup or testTeardown methods, but some of the things that really are just doing heavy lifting utility. this yaml parsing, copyFile, prepWorkingDir, random whatever...

Reply:
Hmm, how about `keybaseTestConfig`? I'd like to keep the config part of the name the same because that's what the js bot lib uses